src/app/controllers/sap_portal.php

```diff
@@ -182,13 +182,17 @@ public function create_mission() {
         }
     }
 
-    public function review_mission($mission_id) {
+    /* $approved specifies whether the approved or unapproved missions have to be shown */
+    public function review_mission($mission_id, $approved = false) {
```

Comment:
If someone goes to `mission/<id>/unapproved`, you'll still show approved submissions! :P